### DISCUSSION TITLE
discord.on_interaction

### DISCUSSION THREAD

**Discussion by PyMohsen** [Pycord-Development/pycord#779] [Q&A] [answered]
How can I use `discord.on_interaction` event?
Please say example

**Comment by VincentRPS** [comment]
Just read the docs, it should get you an idea of what to do there. Also you just need to put it in your client.event or client.listen

**Comment by Dorukyum** [comment, marked as answer]
[Read the docs.](https://docs.pycord.dev/en/master/api.html?highlight=on_interaction#discord.on_interaction)

**Comment by Lulalaby** [comment]
You are literally on our discord. Ask there

**Comment by ChickenDevs** [comment]
Is it that difficult to open discord and click on the blurple and white snaky boi?
